qemu/tests/timerdevice_time_jump_check.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from virttest import error_context
+from virttest.compat_52lts import decode_to_text
+
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    check time jumps in guest (only for Linux guest):
+
+    1) boot guest with '-rtc base=utc,clock=host,driftfix=slew'
+    2) check current clocksource in guest
+    3) pin all vcpus to specfic host CPUs
+    4) verify time jump
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+
+    error_context.context("Check the clock source currently used on guest",
+                          logging.info)
+    cmd = "cat /sys/devices/system/clocksource/"
+    cmd += "clocksource0/current_clocksource"
+    logging.info("%s is current clocksource." % session.cmd_output(cmd))
```

Comment:
Is this check meaningful besides a log info?

Reply:
@vivianQizhu Add this for debugging.